### PR TITLE
Update V2 components + Move to react-icons

### PR DESCRIPTION
**Pull request by sean-brydon**
## What does this PR do?

Moves from react-feather to react-icons/fi

Includes a few updates to V2 components to merge into main - to allow @baileypumfleet to start working with V2 on console :) 

Also fixes 
image
[Zero appearing while using `&&`](https://www.designcise.com/web/tutorial/why-is-react-showing-0-when-conditionally-rendering-a-component
)

- [X] Bug fix (non-breaking change which fixes an issue)
- [] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

### CODE REVIEW

File: apps/web/pages/event-types/index.tsx [hunk 1]
```diff
@@ -69,9 +69,7 @@ const Item = ({
 }) => {
   const { t } = useLocale();
 
-  function isCalendarConnectedMissing() {
-    return connectedCalendars?.length && !type.team && !type.destinationCalendar;
-  }
+  const isCalendarConnectedMissing = connectedCalendars?.length && !type.team && !type.destinationCalendar;
```
Comment (sean-brydon) [inline apps/web/pages/event-types/index.tsx:72]: Fixes 0 issue appearing. Don't think this needed to be a function as it was being called on render anyway.

File: packages/ui/v2/AvatarGroup.tsx [hunk 2]
```diff
@@ -18,19 +18,16 @@ export const AvatarGroup = function AvatarGroup(props: AvatarGroupProps) {
   const LENGTH = props.items.length;
 
   return (
-    <ul className={classNames(props.className)}>
+    <ul className={classNames(props.className, "flex items-center")}>
```
Comment (sean-brydon) [inline packages/ui/v2/AvatarGroup.tsx:21]: Fixes last avatar in group floating into space

File: packages/ui/v2/Button.tsx [hunk 3]
```diff
@@ -80,7 +82,7 @@ export const Button = forwardRef<HTMLAnchorElement | HTMLButtonElement, ButtonPr
       ref: forwardedRef,
       className: classNames(
         // base styles independent what type of button it is
-        "inline-flex items-center text-sm font-medium",
+        "inline-flex justify-center items-center text-sm font-medium",
```
Comment (sean-brydon) [inline packages/ui/v2/Button.tsx:85]: Always centre text

File: packages/ui/v2/EmptyScreen.tsx [hunk 4]
```diff
@@ -11,25 +11,28 @@ export default function EmptyScreen({
...
+          <Icon className="inline-block h-10 w-10 text-white dark:bg-gray-900 dark:text-gray-600" />
         </div>
         <div className="max-w-[420px] text-center">
           <h2 className="text-semibold font-cal mt-6 text-xl dark:text-gray-300">{headline}</h2>
-          <p className="mt-4 mb-8 text-sm font-normal leading-6 text-gray-700 dark:text-gray-300">
+          <p className="line-clamp-2 mt-3 mb-8 text-sm font-normal leading-6 text-gray-700 dark:text-gray-300">
             {description}
           </p>
           {buttonOnClick && buttonText && <Button onClick={(e) => buttonOnClick(e)}>{buttonText}</Button>}
+          {buttonRaw}
```
Comment (sean-brydon) [inline packages/ui/v2/EmptyScreen.tsx:35]: Have the ability to render a custom button to match one off designs

File: packages/ui/v2/booker/DatePicker.tsx [hunk 5]
```diff
@@ -83,7 +83,7 @@ const Days = ({
             <div key={`e-${idx}`} />
           ) : props.isLoading ? (
             <button
-              className="absolute top-0 left-0 right-0 bottom-0 mx-auto flex w-full items-center justify-center rounded-sm border-transparent bg-gray-50 text-center text-gray-400 opacity-50 dark:bg-gray-900 dark:text-gray-400"
```
Comment (sean-brydon) [inline packages/ui/v2/booker/DatePicker.tsx:86]: Prettier

File: packages/ui/v2/form/Select.tsx [hunk 6]
```diff
@@ -1,111 +1,62 @@
...
+  return (
+    <components.Input
+      // disables our default form focus hightlight on the react-select input element
+      inputClassName={classNames("focus:ring-0 focus:ring-offset-0", inputClassName)}
+      {...props}
+    />
+  );
+};
 
+function Select<
```
Comment (sean-brydon) [inline packages/ui/v2/form/Select.tsx:24]: Use old style of select - was trying to make this integrate nicely with tailwind but we may aswell just use react-select.

**Review by pumfleet — APPROVED** [review]
Code looks good to me

**Pull request merged**